Examples/socketio-python/README.md

```diff
@@ -0,0 +1,79 @@
+## Python Socketio Example
+
+This example illustrates one way to utilize socketio for data transmittal.
+
+There are 3 files:
+
+`client_logger.py`
+- Socketio client that listens for event "log_telemetry" and logs
+the data packet to console
+
+`client_telemetry.py`
+- Socketio client that generates telem (simulates receiving from the TBM) and emits a "log_telemetry" event containing the data packet
+
+`server.py`
+- Standard socketio server, handles event routing between clients  
```

Comment:
Does not really matter that much, but should we mention now that this is flask-socketio?

Reply:
True - resolved